maven-plugin/src/main/resources/build-distro/web/Dockerfile-tomcat8-jre11

```diff
@@ -0,0 +1,23 @@
+FROM tomcat:8-jre11
```

Comment:
It would probably be best if we could replace these Dockerfiles with something based on openmrs-core, probably not too dissimilar from what we do in the distro-referenceapplication repo: https://github.com/openmrs/openmrs-distro-referenceapplication/blob/main/Dockerfile (I'd actually like to swap out the Dockerfile there for a version built by the SDK).

Reply:
Are we also replacing the existing docker files? https://github.com/openmrs/openmrs-sdk/tree/master/maven-plugin/src/main/resources/build-distro/web

Reply:
@ibacher, @dkayiwa any insights on how we should do this??

Reply:
I'm just imaging having a Dockerfile like:

```docker
FROM openmrs/openmrs-core:nightly-amazoncorretto-11

COPY openmrs.war /openmrs/distribution/openmrs_core
COPY modules /openmrs/distribution/openmrs_modules
COPY owa /openmrs/distribution/openmrs_owa
```

We'd also have to likely update the script [here](https://github.com/openmrs/openmrs-core/blob/master/startup-init.sh) to handle the O3 SPa.